src/datasources/__tests__/subscription.test.js

```diff
@@ -34,7 +34,7 @@ describe('subscription#fetch', () => {
             alias: 'items',
             args: {},
             fieldsByTypeName: {
-              Subscription: {
+              CMRSubscription: {
```

Comment:
Totally spitballing here. But would it be better to call this `SearchSubscription`? Maybe slightly more inline with the other types.

Reply:
Yeah I'm fine changing this. It's a bummer we are going to have this outlier, but the auto-gen documentation will be better renaming this